### DISCUSSION TITLE
Even though the live streaming was stopped and resumed, the recording could not be restarted and ended.(pandalive.py)

### DISCUSSION THREAD

**Discussion by KOONWHITE** [streamlink/streamlink#5275] [Q&A] [answered]
### Description

```
streamlink --http-cookie "3be3f8e358abbf54cec643229de77fc9e4f3f0bbc9b171580d45d13aaa374c16=●" --http-cookie "_gid=●" --http-cookie "_ga_NGSHFJTQS1=●" --http-cookie "_ga_W91XDLC3YE=●" --http-cookie "_gcl_au=●" --http-cookie "_ga=●" --http-cookie "sessKey=●" --http-cookie "userLoginIdx=●" --http-cookie "userLoginSaveID=●" "●" best -o "a.ts"
```

Hello. I have a problem.
Recording the broadcast by entering the options above works well at first.

However, due to the streamer's network problem, the live streaming is interrupted and when it restarts, the recording ends without restarting. (404 Client Error: Not Found for url)
So I tried adding options like --stream-segment-threads, --stream-segment-attempts, and --stream-segment-timeout, but the problem is still not solved.

When live streaming is interrupted and restarted, the original m3u8 address disappears and a new m3u8 address is created.
I think it's a problem caused by not being able to catch the newly created m3u8 address.

What should I do? I ask for your help.

**Comment by bastimeyer** [comment, marked as answer]
> What should I do?

Restart Streamlink. There's nothing that can be done. Once the stream URL has been resolved by the plugin, Streamlink needs to be restarted if the streaming site requires a different stream URL when something changes on their end. Some plugins have custom logic that can update the resolved stream URL or its parameters, like expiration time parameters, etc.

**Reply by KOONWHITE** [reply]
Can you tell me which plugin has the custom logic that can update the resolved stream URL or its parameters, like expiration time parameters, etc?
I'm trying to fix the pandalive plugin by asking chatgpt, but I still can't find a clear answer.

**Reply by bastimeyer** [reply]
> I'm trying to fix the pandalive plugin by asking chatgpt

lol

> which plugin has the custom logic that can update the resolved stream URL

Something like this does only work for streams which **require** an update of a stream URL **while the stream is continuing**. For sites where the channel has gone offline and come back online again, this won't work. As said, restart Streamlink. If you're trying to record a stream, then write a wrapper shell script that loops Streamlink. Good luck.

**Reply by KOONWHITE** [reply]
My question was vague. The channel does not go offline, but is in standby for a while.
It seems that only new m3u8 addresses are provided when the network is reconnected.
Is it still just a way to restart streamlink?
